src/Parser/Shaman/Elemental/Modules/Items/TheDeceiversBloodPact.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import ItemIcon from 'common/ItemIcon';
+import ItemLink from 'common/ItemLink';
+import { formatNumber } from 'common/format';
+
+
+class TheDeceiversBloodPact extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  extraMaelstrom = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.THE_DECEIVERS_BLOOD_PACT_EQUIP.id);
```

Comment:
You could use the `.hasFeet` function to also check the item ID, and enable based on that.